jmclient/jmclient/blockchaininterface.py

```diff
@@ -215,6 +215,39 @@ def get_block(self, blockheight):
             return False
         return block
 
+    def rescanblockchain(self, start_height, end_height=None):
+        # Threading is not used in Joinmarket but due to blocking
+        # nature of this very slow RPC call, we need to fire and forget.
+        from threading import Thread
+        Thread(target=self.rescan_in_thread, args=(start_height,),
+               daemon=True).start()
+
+    def rescan_in_thread(self, start_height):
```

Comment:
```suggestion
    def rescan_in_thread(self, start_height: int) -> None:
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1461/commits/7f4eaa9846f2f489480de1f2f8bfa9a1e2e96686